node/middlewares/checkPublishedApp.ts

```diff
@@ -14,6 +14,11 @@ export async function checkPublishedApp(
 
   const { vbase } = ctx.clients
   const { buildId } = body
+  let { targetWorkspace } = body
+
+  if (targetWorkspace === undefined) {
+    targetWorkspace = ctx.vtex.workspace
+  }
```

Comment:
Can workspace be null here? Why do we need to check for `=== undefined`?

```suggestion
  if (!targetWorkspace) {
    targetWorkspace = ctx.vtex.workspace
  }
```

Reply:
I made it so that if there is no workspace specified, the app will be installed in the current workspace. That way we will not break with the old CMS inputs.